nixos/lib/make-disk-image.nix

```diff
@@ -368,11 +391,27 @@ let format' = format; in let
     diskImage=$out/${filename}
   '';
 
+  createEFIVars = ''
+    efiVars=$out/efi-vars.fd
+    cp ${efiVariables} $efiVars
```

Comment:
I've noticed a number of potential quoting issues around this module (here, line 406, 429, 430, 452, ...)
Can you run the resulting scripts through shellscript and fix them?

Reply:
Of course. :)